CHANGELOG.md

```diff
@@ -60,10 +60,6 @@ The format is based on [Keep a Changelog].
 
 ### Deprecated
 
-- `Backend.jobs()` and `Backend.retrieve_job()` have been deprecated in favour
```

Comment:
Instead of removing the whole comment, can you reword it and move it to `Added`? Don't worry too much about the wording itself - the important bit is having it on the changelog in order to make the changelog cleanup before the release easier.